Sources/Testing/Running/Runner.RuntimeState.swift

```diff
@@ -152,13 +152,14 @@ extension Configuration {
     let configuration = Self._all.withLock { all in
       all.instances.removeValue(forKey: id)
     }
-    if let configuration, configuration.deliverExpectationCheckedEvents {
+    if let configuration, configuration.eventHandlingOptions.isExpectationCheckedEventEnabled {
       Self._deliverExpectationCheckedEventsCount.decrement()
     }
   }
 
   /// An atomic counter that tracks the number of "current" configurations that
-  /// have set ``deliverExpectationCheckedEvents`` to `true`.
+  /// have set ``EventHandlingOptions/isExpectationCheckedEventEnabled`` to
+  /// `true`.
```

Comment:
Mind deleting the "On older platforms" blurb here? It's not accurate (I was trying to use `Atomic` but ran into technical issues.)